ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/uplink_lan_wan_router2.cfg

```diff
@@ -117,13 +117,15 @@ interface Ethernet2.10
    no shutdown
    mtu 9214
    encapsulation dot1q vlan 10
+   flow tracker hardware FLOW-TRACKER
    vrf VRF1
    ip address 10.0.10.1/24
 !
 interface Ethernet2.100
    description My vlan 100
    no shutdown
    encapsulation dot1q vlan 100
+   flow tracker hardware FLOW-TRACKER
```

Comment:
so is this back enabled by default on sub interfaces? thats probably linked to the comment I made below on the sub interfaces

Reply:
Cmmented out the sub intf tracking for now, will enable after resolution of the other comment.